beam/www/beam/scan.ts

```diff
@@ -47,8 +47,9 @@ export class ScanHandler {
 
 	add_or_associate(barcode_context: FormContext[]) {
 		barcode_context.forEach(async action => {
+			const parentfield = action.parentfield || 'items'
 			const is_stock_entry =
-				this.store.form.doctype === 'Stock Entry' &&
+				(this.store.form.doctype === 'Stock Entry' || 'Work Order') &&
```

Comment:
@agritheory I don't think the string change (+ 'Work Order') will do anything since both conditions in this check need to be satisfied for the variable to be truthy, and the second check will never be true for Work Order since the `stock_entry_type` field doesn't exist on WO.

Reply:
I was trying to unblock the reactivity of the items table via scan, but I think you've gotten that figured out elsewhere.